### PR TITLE
update sunjayBhatia affiliation

### PR DESCRIPTION
**Pull request by sunjayBhatia**
Originally updated here: https://github.com/cncf/gitdm/pull/432 but corrected since that change seems to have been overwritten

### CODE REVIEW

File: developers_affiliations4.txt [hunk 1]
```diff
@@ -17828,8 +17828,9 @@ suniltheta: suniltheta!users.noreply.github.com
 	Amazon from 2021-03-01
 sunishvk: sunish.vk!cognizant.com
 	Cognizant
-sunjayBhatia: sbhatia!pivotal.io
-	Pivotal
+sunjayBhatia: sbhatia!pivotal.io, sunjayb!vmware.com, sunjayBhatia!users.noreply.github.com
+	Pivotal Software, Inc. until 2019-12-30
+	VMware, Inc. from 2019-12-30
```
Comment (sbueringer) [inline developers_affiliations4.txt]:
```suggestion
	VMware Inc. from 2019-12-30
```
Not sure if it makes a difference

Comment (sunjayBhatia) [reply developers_affiliations4.txt]: I think the name is officially documented with the comma

Comment (sbueringer) [reply developers_affiliations4.txt]: @sunjayBhatia In those two places it's without the comma:
* https://github.com/cncf/gitdm/blob/master/company_developers6.txt#L4611
* https://k8s.devstats.cncf.io/d/9/companies-table?orgId=1&var-period_name=Last%20quarter&var-metric=contributions
(that's why I assume it's correct without, I don't have any additional infos/knowledge :))

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm, sorry for late answer but I'm on PTO.

**Pull request merged**